packages/fcl/src/wallet-utils/on-fcl-message.js

```diff
@@ -0,0 +1,11 @@
+export const onFclMessage = (msg, cb = () => {}) => {
```

Comment:
Can we rename this to something like `onMessageFromFCL`? Just so the two methods, this one and `sendMsgToFCL` have the `FCL` portion capitalized?

Reply:
I like this and have updated to `onMessageFromFCL`